.changeset/long-swans-drive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/cli': major
```

Comment:
I dont think this is a major change actually?

Reply:
You are right I'll change it to minor